index.html

```diff
@@ -1,85 +1,117 @@
+    <h2>A <div id="frameworks">browserside</div> implementation of the Magister 6 API</h2>
+  </div>
+
+  <div class="main">
+
+
+    <!--// Introduction. //-->
+
+    Magister.js is an opensource asynchronous JavaScript implementation of the non-public API of <a href="http://magister6.nl" target="_blank">Magister</a>, built by <a href="http://lieuwex.github.io/" target="_blank">Lieuwe</a> @ <a href="http://simplyapps.nl/" target="_blank">simplyApps</a>.<br>
+    The goal is to create an easy to use and open way to communicate with Magister.<br><br>
+    Help by making pull requests and issues at <a href="https://github.com/simplyGits/magisterJS">the Magister.js repo</a>.<br>
+    Code is <a href="docs/index.html">documented</a> with YUIDoc.<br><br>
+
+    <a href="tryout/index.html">Try Magister.js in your browser!</a><br><br>
+
+
+    <!--// Getting Started. //-->
+
+    <div class="subHeading">Get started</div>
+
+    <div class="platformTitle">Node.js</div>
+    <pre><code class="language-markup" >npm install magister.js</code></pre>
+
+    <div class="platformTitle">Meteor</div>
+    <pre><code class="language-markup" >meteor add simply:magisterjs</code></pre>
+    <div class="platformTitle">Browserside</div>
+    <pre class="language-markup"><a href="https://github.com/simplyGits/MagisterJS/raw/master/Magister.js-browser.zip" download>Download minified JS with README</a></pre>
+
+
+    <!--// Examples. //-->
+
+    <br><br><div class="subHeading">Examples</div><br>
+
+    <div class="codeExampleDescription" style="margin-top: 5px;">Require magister.js (for node.js)</div>
+    <pre><code class="language-javascript">var Magister = require('magister.js');</code></pre>
+
+    <div class="codeExampleDescription">Print the full name of the teacher for the first lesson of today.</div>
+    <pre><code class="language-javascript">new Magister.Magister({
+      school: 'xxxx',
+      username: 'xxxx',
+      password: 'xxxx'
+    }).ready(function (error) {
+      if (error) {
+        return console.log('oops!', error);
+      }
+      this.appointments(new Date(), function (error, result) {
+        console.log(result[0].teachers()[0].fullName());
+      });
+    });</code></pre>
+
+    <div class="codeExampleDescription">Send a message to my 'friends', I.. I... don't have friends..</div>
+    <pre><code class="language-javascript">new Magister.Magister({
+      school: 'xxxx',
+      username: 'xxxx',
+      password: 'xxxx'
+    }).ready(function (error) {
+      if (error) {
+        return console.log('oops!', error);
+      }
+      this.composeAndSendMessage("subject", "body", ["Thomas", "Tom", "Jesse"]);
+    });</code></pre>
+
+    <div class="codeExampleDescription">Print recent grades.</div>
+    <pre><code class="language-javascript">new Magister.Magister({
+      school: 'xxxx',
+      username: 'xxxx',
+      password: 'xxxx'
+    }).ready(function (error) {
+      if (error) return console.log('oops!', error);
```

Comment:
It's [not exactly](https://dorey.github.io/JavaScript-Equality-Table/) equivalent, but since this is only going to be `null` or an object, it's fine of course. Since this isn't my code, it isn't my call to give a style advice... but personally I'd go with `if (error)`. Though lately I've been trying out the other style as well. I don't know. But since `if (error)` is 8 chars shorter, go with that one.